modules/oe_content_publication/config/post_updates/00003_create_fields/field.field.node.oe_publication.oe_departments.yml

```diff
@@ -0,0 +1,34 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.oe_departments
+    - node.type.oe_publication
+  module:
+    - rdf_skos
+_core:
+  default_config_hash: l0I7G6TgMBCSa_rCRA5P2qly9lFT7xc2KoBk364ERhs
```

Comment:
This should be removed.

Reply:
@22Alexandra this is added as requested by @ademarco